tracker/tracker/src/main/app/index.ts

```diff
@@ -323,7 +323,14 @@ export default class App {
 
     const projectPath = isSaas ? ingest.replace('api', 'app') : ingest
 
-    return projectPath.replace(/ingest$/, `${projectID}/session/${sessionID}`)
+    const url = projectPath.replace(/ingest$/, `${projectID}/session/${sessionID}`)
+
+    if (options?.jumpToCurrentTime) {
+      const jumpTo = Date.now() - timestamp
```

Comment:
actually, you can use `now()` from `src/main/utils.ts` here (should be more precise)

Reply:
Yes, please, use `now()` - it is already exported. Otherwise, all the time  adjustments (with backend, for example) won't be taken into consideration